src/Spatial/Euclidean/Quaternion.cs

```diff
@@ -152,7 +140,7 @@ public bool IsUnitQuaternion
         /// </summary>
         public bool IsRotation
```

Comment:
But does this different numerical criterion actually make sense? I.e. does it make sense that a specific quaternion can be at the same time considered as unit quaternion but not as rotation?